central/graphql/resolvers/service_accounts_test.go

```diff
@@ -176,7 +176,7 @@ func (s *ServiceAccountResolverTestSuite) TestGetSaNamespace() {
 
 func (s *ServiceAccountResolverTestSuite) getMockContext(extraPerms ...permissions.ResourceMetadata) context.Context {
 	id := mockIdentity.NewMockIdentity(s.mockCtrl)
-	id.EXPECT().UID().Return(fakeUserID).AnyTimes()
```

Comment:
This was apparently defined in `central/graphql/resolvers/vulnerability_requests_test.go` so it won't compile for non-pg tests.